src/interface/meshblock_data.cpp

```diff
@@ -338,25 +342,24 @@ MeshBlockData<T>::PackVariablesAndFluxes(const std::vector<MetadataFlag> &flags)
 ///        indices represent inclusive bounds for, e.g., a sparse or tensor-valued
 ///        variable.
 template <typename T>
-VariablePack<T>
-MeshBlockData<T>::PackVariablesHelper_(const std::vector<std::string> &names,
-                                       const vpack_types::VarList<T> &vars,
-                                       PackIndexMap &vmap) {
-  // this->GetSparseMap
-  auto kvpair = varPackMap_.find(names);
+PackVariablesResult<T>
+MeshBlockData<T>::PackListedVariables(VariableListResult &&variables) {
```

Comment:
Changing the naming here and removing the "Helper" functions seems to make this a more public API. Is that intentional?

Reply:
It's declared private, so it's not intended to be a public API. But at the same time, the `Helper_` suffix seems unnecessary to me.